packages/components/src/components/primitives/Progress/types.ts

```diff
@@ -0,0 +1,8 @@
+import { ProgressProps as ChakraProgressProps } from '@chakra-ui/progress';
+
+export const progressVariants = ['primary', 'second'] as const;
+export type ProgressVariant = typeof progressVariants[number];
+
+export interface ProgressProps extends ChakraProgressProps {
+  variant?: ProgressVariant;
+}
```

Comment:
Where have you seen these `variants`? I haven't been able to find this in the figma file.

I've seen that the `Progress` could have different sizes that I can't see defined in here. (xs, s, m, l, xl). Could you review the figma file and apply these changes please?